ai-cfia-ia-acia/docs/about/team/cs.md

```diff
@@ -2,4 +2,60 @@
 sidebar_position: 2
 ---
 
-# Computer Science Team
\ No newline at end of file
+# Computer Science Team
```

Comment:
I think Computer Science is probably a misnomer due to the old classification between "CS". 

The classification itself is being renamed IT which is a more accurate name of what we actually do: 

https://pipsc.ca/groups/cs/cs-group-conversion-to-it-group

Even more specifically here, we should probably be the "Software Engineernig Team" as it more accurately defines what we actually do.